qa/tasks/mgr/dashboard/helper.py

```diff
@@ -339,12 +339,12 @@ def _request(cls, url, method, data=None, params=None, version=DEFAULT_API_VERSI
 
     @classmethod
     def _get(cls, url, params=None, version=DEFAULT_API_VERSION, set_cookies=False, headers=None,
-             retries=0, wait_func=None):
+             retries=1, wait_func=None):
         while retries >= 0:
             try:
                 return cls._request(url, 'GET', params=params, version=version,
                                     set_cookies=set_cookies, headers=headers)
-            except requests.RequestException as e:
+            except (Exception, requests.RequestException) as e:  # pylint: disable=broad-except
```

Comment:
I don't know if its good to change the below to extend from `requests.RequestException` is a good idea. So I went for catching a broader exception. https://github.com/ceph/ceph/blob/ed87297e31d63f98fa68c0de0caaf22b78401269/src/pybind/mgr/dashboard/rest_client.py#L50

Reply:
Weird. These are the [`requests` Exceptions](https://requests.readthedocs.io/en/latest/_modules/requests/exceptions/). In this case, I'd remove the second one, which is redundant being a subclass of the former.